kubernetes/.swagger-codegen/VERSION

```diff
@@ -1 +1 @@
-2.4.0-SNAPSHOT
\ No newline at end of file
+2.3.0-SNAPSHOT
```

Comment:
Is this correct given that you used the 3.0.0  branch?

Reply:
Yeah, that's what it generated. I have no idea why :)